rts/lra/lra-coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/Transaction.java

```diff
@@ -790,15 +832,34 @@ public boolean afterLRANotification() {
 
                 while (listeners.hasNext()) {
                     URI uri = listeners.next();
+                    Response response = null;
+
+                    try {
+                        Future<Response> asyncResponse = client.target(uri)
+                                .request()
+                                .header(LRA.LRA_HTTP_ENDED_CONTEXT_HEADER, id)
+                                .async()
+                                .put(Entity.text(status.name()));
+
+                        response = asyncResponse.get(PARTICIPANT_TIMEOUT, TimeUnit.SECONDS);
```

Comment:
Because the expectation is that the network call should complete within milliseconds. Setting the value to 1000 is just protection against an unavailable server locking up the calling thread indefinitely. The network call will be reattempted on the next period recovery pass so it is transactionally safe to do this.

Reply:
ok, I see